### PR TITLE
chore: Move distinct and active code_of_conduct file to /docs

### PR DESCRIPTION
**Pull request by Ana2k**
fixes: #355 
fixes: #356

Description:
Merged both code of conduct md files into one. Shifted the merged file into docs folder.

### CODE REVIEW

**Comment by jingtang10** [comment]
Thanks for the PR @Ana2k. Appreciate it. As described in here: https://opensource.google/conduct/ we have adopted the more explicit version of the code of conduct. So I think you don't need to actually merge the two. You can overwrite the one in `/docs`.

Please also rename the PR to reflect this (e.g. remove redundant code of conduct file and move the active one to /docs or something similar)

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -90,4 +90,70 @@ harassment or threats to anyone's safety, we may take action without notice.
...
+taken. The identity of the reporter will be omitted from the details of the
+report supplied to the accused. In potentially harmful situations, such as
+ongoing harassment or threats to anyone's safety, we may take action without
+notice.
+
+*This document was adapted from the [IndieWeb Code of Conduct][] and can also
+be found at <https://opensource.google/conduct/>.*
+
+[IndieWeb Code of Conduct]: https://indieweb.org/code-of-conduct
+
```
Comment (jingtang10) [inline docs/CODE_OF_CONDUCT.md:159]: just remove this.

Comment (Ana2k) [reply docs/CODE_OF_CONDUCT.md:159]: @jingtang10 ok sure I will do this ASAP. Thanks for the specifics.

**Review by jingtang10 — CHANGES_REQUESTED** [review]
please update once done

**Comment by Ana2k** [comment]
> please update once done

@jingtang10 I have made the suggested changes, please review them once you are free.

**Review by jingtang10 — APPROVED** [review]
Thanks for this @Ana2k! 🎉🚀

**Pull request merged**